### ISSUE TITLE
Crash on fast alt-tab

### ISSUE THREAD

**Issue by mattboll** [fyne-io/fynedesk#122]
If I press alt-tab to change window and release the keys fast enough, I've got the following stacktrace

```
2020/06/30 22:29:34   At: /home/mbollot/github/fynedesk/internal/x11/win/property.go:69
goroutine 1 [running, locked to thread]:
runtime/debug.Stack(0x7e4be1, 0x966c10, 0x438e40)
	/usr/lib/go-1.14/src/runtime/debug/stack.go:24 +0x9d
runtime/debug.PrintStack()
	/usr/lib/go-1.14/src/runtime/debug/stack.go:16 +0x22
fyne.io/fynedesk/internal/ui.(*desktop).runFull.func1(0xc0001c4000)
	/home/mbollot/github/fynedesk/internal/ui/desk_full.go:23 +0x42
panic(0xa242a0, 0xfadce0)
	/usr/lib/go-1.14/src/runtime/panic.go:969 +0x166
github.com/go-gl/glfw/v3.3/glfw.(*Window).SetTitle.func2(0x0, 0x2565c90)
	/home/mbollot/go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20200222043503-6f7a984d4dc4/window.go:393 +0x22
github.com/go-gl/glfw/v3.3/glfw.(*Window).SetTitle(0x0, 0xad716e, 0x10)
	/home/mbollot/go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20200222043503-6f7a984d4dc4/window.go:393 +0x84
fyne.io/fyne/internal/driver/glfw.(*window).doShow.func1()
	/home/mbollot/go/pkg/mod/fyne.io/fyne@v1.3.0/internal/driver/glfw/window.go:340 +0x7b
fyne.io/fyne/internal/driver/glfw.(*gLDriver).runGL(0xc0000d43c0)
	/home/mbollot/go/pkg/mod/fyne.io/fyne@v1.3.0/internal/driver/glfw/loop.go:96 +0x59b
fyne.io/fyne/internal/driver/glfw.(*gLDriver).Run(0xc0000d43c0)
	/home/mbollot/go/pkg/mod/fyne.io/fyne@v1.3.0/internal/driver/glfw/driver.go:70 +0x37
fyne.io/fyne/internal/driver/glfw.(*window).ShowAndRun(0xc0001c8000)
	/home/mbollot/go/pkg/mod/fyne.io/fyne@v1.3.0/internal/driver/glfw/window.go:387 +0x57
fyne.io/fynedesk/internal/ui.(*desktop).runFull(0xc0001c4000)
	/home/mbollot/github/fynedesk/internal/ui/desk_full.go:28 +0x6e
fyne.io/fynedesk/internal/ui.(*desktop).Run(0xc0001c4000)
	/home/mbollot/github/fynedesk/internal/ui/desk.go:158 +0x54
main.main()
	/home/mbollot/github/fynedesk/cmd/fynedesk/main.go:15 +0xfb

```

**Comment by andydotxyz** [comment]
I think this is related to fyne-io/fyne#1116 which is already fixed ready for 1.3.1 I suspect that once we update to that release this will go away :) 🤞

**Comment by andydotxyz** [comment]
I upgraded develop branch to the fyne nearly-released develop so we can test.
This seems to be resolved :)
